frontend/amundsen_application/static/js/pages/HomePage/index.tsx

```diff
@@ -27,13 +27,57 @@ export interface DispatchFromProps {
   searchReset: () => UpdateSearchStateReset;
 }
 
+export type Widget = {
+  // TODO: What file does the Widget type belong in?
+  name: string;
+  options: {
+    path: '/';
+  };
+};
+
+export type HomePageLayout = Widget[];
+
 export type HomePageProps = DispatchFromProps & RouteComponentProps<any>;
 
 export class HomePage extends React.Component<HomePageProps> {
   componentDidMount() {
     this.props.searchReset();
   }
 
+  getHomePageWidgets(layout: HomePageLayout): React.ReactNode[] {
+    /* TODO: Make 100% sure it returns the pre-existing layout absent a custom config,
+    to not break any OSS users' Amundsen implementations */
+
+    return [
+      <div className="home-element-container">
+        <SearchBar />
+      </div>,
+      <div className="filter-breadcrumb pull-right">
+        <Breadcrumb
+          direction="right"
+          path="/search"
+          text={SEARCH_BREADCRUMB_TEXT}
+        />
+      </div>,
```

Comment:
I agree conceptually but would like to leave that change until a little later if you don't mind--it'll require some tweaking beyond just moving the Breadcrumb to be inside a different render, since elsewhere in the UI we render the search bar without the "Advanced Search" Breadcrumb, for example on the advanced search page itself ('/search' URL):

![Screenshot 2022-12-08 at 7 27 18 PM](https://user-images.githubusercontent.com/55849024/206594885-8c398bf0-8448-44b1-8def-9b7a85cec5a6.png)

(screenshot from our prod)